bcda/web/router_test.go

```diff
@@ -128,11 +128,11 @@ func (s *RouterTestSuite) TestEOBExportRoute() {
 	res = s.getAPIRoute("/api/v1/Patients/$export?_type=ExplanationOfBenefit")
 	assert.Equal(s.T(), http.StatusNotFound, res.StatusCode)
 
-	// ALR Patient
-	res = s.getAPIRoute("/api/v1/alr/Patient/$export?_type=ExplanationOfBenefit")
+	// ALR
+	res = s.getAPIRoute("/api/v1/alr/$export?_type=ExplanationOfBenefit")
```

Comment:
Since we don't need to test the ALR endpoint with each resource (EoB, Patient, Coverage) here, we could move it out of these three and into its own section (just the one endpoint for now w/o any `type` filters); will be easier to add to once we start building out more filtering capability, too